sparql/sparql12/syntax-triple-terms-positive/manifest.ttl

```diff
@@ -0,0 +1,608 @@
+@prefix :       <http://www.w3.org/2009/sparql/docs/tests/data-sparql11/syntax-query/manifest#> .
```

Comment:
This prefix should be changed to:
- be 1.2-specific
- change "syntax-query" to "syntax-triple-terms-positive"